### PR TITLE
Oj 1400 deploy fe to build and staging using secure pipelines

### PR DESCRIPTION
**Pull request by Connor-Johnson-DCO**
### What changed

Can deploy this front end via secure pipelines.

Front end can be viewed via: https://review-toy.dev.account.gov.uk/

### Why did it change

To be inline with all other repos for deployment

### Issue tracking

- [OJ-1400](https://govukverify.atlassian.net/browse/OJ-1400)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-1400]: https://govukverify.atlassian.net/browse/OJ-1400?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -19,3 +19,5 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+        types_or: ["javascript", "ts", "json"]
+        args: ["src"]
```
Comment (merlinc) [inline .pre-commit-config.yaml:23]: If this is restricted, then this should include `test` as well.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -19,3 +19,5 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+        types_or: ["javascript", "ts", "json"]
```
Comment (merlinc) [inline .pre-commit-config.yaml:22]: Prettier should be applied to almost all files - its compatible with `.editorconfig` as well.

What was the context for restricting it down to just these files?

File: package.json [hunk 3]
```diff
@@ -27,12 +27,13 @@
     "aws-sdk": "2.1376.0",
     "axios": "^1.4.0",
     "cfenv": "1.2.4",
-    "connect-dynamodb": "3.0.0",
+    "connect-dynamodb": "2.0.5",
```
Comment (merlinc) [inline package.json:30]: Ooh! What happened here?

File: package.json [hunk 4]
```diff
@@ -27,12 +27,13 @@
...
     "di-ipv-cri-common-express": "github:alphagov/di-ipv-cri-common-express#v0.0.36",
     "dotenv": "16.0.3",
     "express": "4.18.2",
     "express-async-errors": "3.1.1",
     "express-session": "1.17.3",
-    "hmpo-app": "3.0.1"
+    "govuk-frontend": "4.3.1",
```
Comment (merlinc) [inline package.json:36]: Ah, damn it, I missed a dependency! :+1:

File: package.json (hunk 3 above)
Comment (Connor-Johnson-DCO) [reply package.json:30]: This was out of sync with some of the other libraries we are using causing a failure, so I reverted it.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (Connor-Johnson-DCO) [reply .pre-commit-config.yaml]: I was seeing what patterns we had in other repos for consistencies sake, but I have removed it now.

**Review by merlinc — DISMISSED** [review]
Could you remove the two `prettier` changes as they're part of the other PR.

Could you also squash your changes, as there's a lot of noise with multiple commits for a small change.

Rest of it looks good to go

**Review by mrdoodles — APPROVED** [review]

**Pull request merged**